configuration/messageproperties/messages_zl_fr.properties

```diff
@@ -29,6 +29,9 @@ zl.status.hivTb.inhCompletionDate=Date d'achèvement de l'INH
 zl.status.hivTb.inhComplete=Complété
 zl.status.hivTb.inhNotComplete=Non complété
 zl.status.hivTb.inhNotStarted=Non initié
+zl.status.regimen.lastPrescribedMedication=Dernier médicament prescrit
+zl.status.regimen.lastDispensedMedication=Dernier médicament distribué
+zl.status.regimen.lastDispensedMedication.none=Aucun
```

Comment:
Use an existing message property -- pihcore.none.label .  Not necessary to create anything new for "none".